tests/lib/languages/js/source-code/source-code.js

```diff
@@ -4885,60 +4883,61 @@ describe("SourceCode", () => {
 			});
 		});
 
-		it("should include parent reference in step args", () => {
```

Comment:
This isn't the case, so I removed this test. It was additionally checking that `steps.args[0]` is `step.target` but that is already covered by another test below.